src/main/asciidoc/_chapters/hbase_mob.adoc

```diff
@@ -613,6 +613,13 @@ $ hdfs dfs -count /hbase/mobdir/data/default/some_table
 This data is spurious and may be reclaimed. You should sideline it, verify your application’s view
 of the table, and then delete it.
 
+==== Data values over the MOB threshold show up stored in the non-MOB hfiles
+
+Bulk load and WAL split-to-HFile feature write data into the normal /hbase/data directory irrespective of data over than the MOB threshold.
```

Comment:
The last part of this sentence reads wonky. Is it what was intended? Thanks.

Reply:
See above @pankaj72981 ?

Reply:
This PR branch is corrupted in my local, have created another PR#2505, Pls check.